### ISSUE TITLE
What is the bare minimum needed to display editor contents?

### ISSUE THREAD

**Issue by sdraper69** [JiHong88/SunEditor#849]
First, great library, really fantastic, I have found it better than a lot of the other "big names" out there.

## Version
2.41.3

## Additional context

I have a bit of an unusual case where there will be an editor and a viewer and would appreciate some guidance. SunEditor is only a small portion of the overall app.

For the editor user, I am turning off some functionality but keeping the majority of it and I am happy to just use the .min.js file which is >2mb. The editor will be able to put in pictures, video/audio links etc. then it all gets saved away.

The editor can then share his project with someone to view it. They will not be collaborating together or anything like that, it will be a read-only view.

So my question....

For the viewer, I am trying to keep the downloads to a minimum so the >2mb min.js file is far from ideal. I see you have mentioned importing just the plugins you need however since the viewer will only ever view the document, do they still require all the plugins that were used to create it in the first place? They will not require the toolbar, or any modals etc. If the editor user has used an image (for example) will the viewer user still require the image plugin - bearing in mind they will NOT be able to edit it in any way or can it be rendered exactly the same without it?

So basically is there a way I could show the result of a suneditor exactly the same way but with minimal suneditor.js download?

Hopefully, that makes sense, many thanks.

**Comment by JiHong88** [comment]
Hi @sdraper69 
When viewing only the document, all you need is the "suneditor-contents.css" file.
Please check this [page](https://github.com/JiHong88/SunEditor#3-contents-display).

**Comment by sdraper69** [comment]
great, really sorry I missed that!

**Comment by sdraper69** [comment]
One further question regarding this, what would be the recommended way for storing and reloading the content?

I was going to do `editor.getContents()` and store that and I can restore that with an  `editor.setContents()` however clearly to set the contents I would require the .min.js - fine for my "editor" user. but not for my "viewer" user.

I feel I am missing something obvious here, so apologies, but how would I store/save the editor and then restore it in "view mode" without having the editor.js loaded. Do I just need to dump the result of the getContents in a div, and assuming the css is loaded it will look the same? Does the div need any particular classes (i can see .se-wrapper seems to hold everything when editing). ANy guidance is really appriciated.
